objectherkenning_openbare_ruimte/performance_evaluation_pipeline/metrics/metrics_utils.py

```diff
@@ -0,0 +1,189 @@
+from enum import Enum
+from typing import Dict, List, Tuple, Union
+
+import numpy as np
+import numpy.typing as npt
+
+
+class ObjectClass(Enum):
```

Comment:
Lovely thing, shall this not be shared and used across all the pipelines in order to avoid possible mismatch between classes?

Reply:
That would be great, but perhaps better for a seperate PR as it will impact more code?